api/tests/integration/infrastructure/repositories/campaign-participation-overview-repository_test.js

```diff
@@ -14,209 +13,173 @@ describe('Integration | Repository | Campaign Participation Overview', () => {
-        expect(campaignParticipationOverviews[0].createdAt).to.deep.equal(new Date('2000-07-01T10:00:00Z'));
-        expect(campaignParticipationOverviews[0].assessmentState).to.equal('completed');
-        expect(campaignParticipationOverviews[0].organizationName).to.equal('1 - My organization');
+          expect(campaignParticipationOverviews[0].id).to.equal(onGoingParticipation.id);
+          expect(campaignParticipationOverviews).to.have.lengthOf(1);
+        });
       });
-    });
 
-    context('with many campaign participations of type `ASSESSMENT`', () => {
-      beforeEach(async () => {
-        databaseBuilder.factory.buildCampaignParticipationElementsForOverview({
-          userId,
-          index: '1',
-          isShared: false,
-          lastAssessmentState: Assessment.states.STARTED,
-          campaignParticipationCreatedAt: new Date('2000-07-01T10:00:00Z'),
-        });
+      context('the filter is TO_SHARE', () => {
+        it('returns participation with a completed assessment', async () => {
+          const states = ['TO_SHARE'];
+          const { campaignParticipationOverviews } = await campaignParticipationOverviewRepository.findByUserIdWithFilters({ userId, states });
 
-        databaseBuilder.factory.buildCampaignParticipationElementsForOverview({
-          userId,
-          index: '2',
-          isShared: false,
-          lastAssessmentState: Assessment.states.COMPLETED,
-          campaignParticipationCreatedAt: new Date('2000-07-02T10:00:00Z'),
+          expect(campaignParticipationOverviews[0].id).to.equal(toShareParticipation.id);
+          expect(campaignParticipationOverviews).to.have.lengthOf(1);
         });
+      });
 
-        databaseBuilder.factory.buildCampaignParticipationElementsForOverview({
-          userId,
-          index: '3',
-          isShared: true,
-          lastAssessmentState: Assessment.states.COMPLETED,
-          campaignParticipationCreatedAt: new Date('2000-07-03T10:00:00Z'),
-          campaignParticipationSharedAt: new Date('2000-07-03T10:00:00Z'),
-        });
+      context('the filter is ENDED', () => {
+        it('returns shared participation or completed participation with an archived campaign', async () => {
+          const states = ['ENDED'];
+          const { campaignParticipationOverviews } = await campaignParticipationOverviewRepository.findByUserIdWithFilters({ userId, states });
 
-        databaseBuilder.factory.buildCampaignParticipationElementsForOverview({
-          userId,
-          index: '4',
-          isShared: false,
-          lastAssessmentState: Assessment.states.STARTED,
-          campaignParticipationCreatedAt: new Date('2000-07-04T10:00:00Z'),
-          campaignArchivedAt: new Date('2000-07-05T11:00:00Z'),
+          expect(campaignParticipationOverviews[0].id).to.equal(endedParticipation.id);
+          expect(campaignParticipationOverviews).to.have.lengthOf(1);
```

Comment:
Il est écrit dans le test: 
`it returns shared participation or completed participation with an archived campaign`

Mais j'ai l'impression qu'on ne teste que la première partie (`returns shared participation `)
Est-ce qu'il ne manque pas un test pour la seconde partie `completed participation with an archived campaign` ?

Reply:
En fait c'est dans le test suivant ^^ du coup il juste réécrire le nom de ce test en `it returns shared participation`

Reply:
done